e2e/inspect/inspect.go

```diff
@@ -25,16 +21,19 @@ type testingEnv struct {
 
 var testenv testingEnv
 
-const expectedLabelsJson = `
+const containerTesterSIF = "testdata/inspecter_container.sif"
+
+const expectedLabelsJSON = `
 {
 	"attributes": {
 		"apps": "",
 		"labels": {
 			"E2E": "AWSOME",
 			"HI": "\"HELLO WORLD\"",
+			"MAINTAINER": "\"WestleyK \u003cwestley@sylabs.io\u003e\"",
```

Comment:
0x3c and 0x3e are `<` and `>` respectively.

It should be possible to use those characters instead of unicode code points.

Also, if you don't want to escape the quotes, you can use backticks. ``"MAINTAINER": `"Westely ..."`,``

Reply:
This output is unmodified output from `singularity inspect inspecter_container.sif`, I did not change anything.

And yes, it does work if I change `\u003c` to `<`.